<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="8" hostname="HPLaptop" failures="10" tests="13" name="Test" time="53.55" errors="0" timestamp="2025-09-27T07:21:28 AEST">
  <testcase classname="api.tests.UserTests" name="createUserTest" time="2.942"/>
  <testcase classname="api.tests.UserTests" name="getUserTest" time="1.271">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.UserTests.getUserTest(UserTests.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- getUserTest -->
  <testcase classname="api.tests.UserTests" name="updateUserTest" time="1.237">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.UserTests.updateUserTest(UserTests.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- updateUserTest -->
  <testcase classname="api.tests.UserTests" name="deleteUserTest" time="1.229">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.UserTests.deleteUserTest(UserTests.java:92)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteUserTest -->
  <testcase classname="api.tests.EmployeeTests" name="createEmployeeTest" time="1.154"/>
  <testcase classname="api.tests.EmployeeTests" name="getEmployeeTest" time="14.464">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at api.tests.EmployeeTests.getEmployeeTest(EmployeeTests.java:90)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- getEmployeeTest -->
  <testcase classname="api.tests.EmployeeTests" name="updateEmployeeTest" time="14.397">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at api.tests.EmployeeTests.updateEmployeeTest(EmployeeTests.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- updateEmployeeTest -->
  <testcase classname="api.tests.EmployeeTests" name="deleteUserTest" time="14.28">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at api.tests.EmployeeTests.deleteUserTest(EmployeeTests.java:154)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteUserTest -->
  <testcase classname="api.tests.ObjectTests" name="getALLObjectsTest" time="0.694"/>
  <testcase classname="api.tests.ObjectDDTests" name="deleteObjectTest" time="0.205">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.ObjectDDTests.deleteObjectTest(ObjectDDTests.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteObjectTest -->
  <testcase classname="api.tests.ObjectDDTests" name="deleteObjectTest" time="0.237">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.ObjectDDTests.deleteObjectTest(ObjectDDTests.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteObjectTest -->
  <testcase classname="api.tests.ObjectDDTests" name="deleteObjectTest" time="0.209">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.ObjectDDTests.deleteObjectTest(ObjectDDTests.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteObjectTest -->
  <testcase classname="api.tests.ObjectDDTests" name="deleteObjectTest" time="0.205">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.ObjectDDTests.deleteObjectTest(ObjectDDTests.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteObjectTest -->
  <testcase classname="api.tests.ObjectTests" name="getObjectTest" time="0.0">
    <ignored/>
  </testcase> <!-- getObjectTest -->
  <testcase classname="api.tests.ObjectTests" name="updateObjectTest" time="0.0">
    <ignored/>
  </testcase> <!-- updateObjectTest -->
  <testcase classname="api.tests.ObjectDDTests" name="getALLObjectsTest" time="0.0">
    <ignored/>
  </testcase> <!-- getALLObjectsTest -->
  <testcase classname="api.tests.ObjectDDTests" name="createObjectTest2" time="0.0">
    <ignored/>
  </testcase> <!-- createObjectTest2 -->
  <testcase classname="api.tests.ObjectTests" name="createObjectTest" time="0.0">
    <ignored/>
  </testcase> <!-- createObjectTest -->
  <testcase classname="api.tests.ObjectTests" name="deleteObjectTest" time="0.0">
    <ignored/>
  </testcase> <!-- deleteObjectTest -->
  <testcase classname="api.tests.ObjectDDTests" name="getObjectTest" time="0.0">
    <ignored/>
  </testcase> <!-- getObjectTest -->
  <testcase classname="api.tests.ObjectDDTests" name="createObjectTest" time="0.0">
    <ignored/>
  </testcase> <!-- createObjectTest -->
</testsuite> <!-- Test -->
